e2e/tests/smoke/file_storage_smoke_test.go

```diff
@@ -42,12 +43,6 @@ func (suite *FileSystemTestSuite) SetupTest() {
 
 	require.NotEmpty(suite.T(), suite.rookTag, "RookTag parameter is required")
 
-}
-
-func (suite *FileSystemTestSuite) TestFileStorage_SmokeTest() {
-	suite.T().Skip("Skipping test : existing issue https://github.com/rook/rook/issues/612")
```

Comment:
let's scope this PR to only the changes needed to get the tests working again. we can save enabling the file tests for another PR